frontend/public/index.html

```diff
@@ -24,7 +24,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>ISCSC - Site du club informatique ISAE-Supaero</title>
```

Comment:
Two things here:
- You should respect the case :wink: 
```diff
- ISCSC
+ iScsc
```
- An english title would be better to be consistent :thinking: